examples/js/loaders/sea3d/SEA3DDraco.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * 	SEA3D - Google Draco
+ * 	@author Sunag / http://www.sunag.com.br/
+ */
+
+'use strict';
+
+//
+//	Lossy Compression
+//
+
+SEA3D.GeometryDraco = function ( name, data, sea3d ) {
+
+	this.name = name;
+	this.data = data;
+	this.sea3d = sea3d;
+
+	var attrib = data.readUShort(),
+		i;
+
+	this.isBig = ( attrib & 1 ) !== 0;
+
+	data.readVInt = this.isBig ? data.readUInt : data.readUShort;
+
+	this.groups = [];
+
+	if ( attrib & 32 ) {
+
+		this.uv = [];
+		this.uv.length = data.readUByte();
+
+	}
+
+	if ( attrib & 1024 ) {
+
+		var numGroups = data.readUByte(),
+			groupOffset = 0;
+
+		for ( i = 0; i < numGroups; i ++ )		{
+
+			var groupLength = data.readVInt() * 3;
+
+			this.groups.push( {
+				start: groupOffset,
+				count: groupLength,
+			} );
+
+			groupOffset += groupLength;
+
+		}
+
+	}
+
+	var module = SEA3D.GeometryDraco.getModule(),
+		dracoData = new Int8Array( data.concat( data.position, data.bytesAvailable ).buffer );
+
+	var decoder = new module.Decoder();
```

Comment:
The Draco decoder has an `isVersionSupported` method that can be used to verify the model was encoded by a version of the library that the current decoder is compatible with — it may be worth storing the Draco version in the asset and checking it here. The Draco library does not guarantee that future versions will be backward compatible (although clearly that is prioritized). The glTF Draco extension is currently locked to a particular version of the Draco library, to ensure long-term compatibility.

Reply:
Thank you for tip! Really without update the `draco libs` this not work.

Reply:
@donmccurdy I believe that if update the `draco_decoder` any version of draco is compatible until then?

Reply:
> I believe that if update the draco_decoder any version of draco is compatible until then?

Do you mean, an up-to-date `draco_decoder` is compatible with files created by any previous version of `draco_encoder`? I believe that is true today. But it is possible that some future version of `draco_decoder` may not support older files, so recording the asset version — or just requiring models be created with a particular encoder version — might be safest.

See https://github.com/google/draco/issues/79.

Reply:
(it's totally fine that you're updating the draco libs here though — [we have a gltf/ subfolder](https://github.com/mrdoob/three.js/tree/dev/examples/js/libs/draco) for the encoder/decoder version required by glTF)

Reply:
I test `isVersionSupported` but this method is out-of-date (I did not understand the reason for this method to be in JS and not in c++ core). The Draco file version is `2.2` this method returns `false`, 1.1 for example returns `true`. I belive that this method work with glTF because fixed version of Draco is bellow of 2. I will consider your tip using `try/catch`, thanks!

https://github.com/google/draco/blob/master/src/draco/javascript/emscripten/version.js

I think that for a reformulate versions of Draco I think of creating a new class, SEA3DDraco2.js or other `class` inside this script.

Reply:
> I will consider your tip using try/catch

Or `decoder.DecodeBufferToMesh( buffer, mesh )`, this method too checks compatibility with versions.

Reply:
Ok, the warning looks good!